src/SourceBuild/patches/nuget-client/0001-Disable-netframework-tfm-trimming.patch

```diff
@@ -11,9 +11,9 @@ diff --git a/build/common.project.props b/build/common.project.props
 index c7f8d32e5..f95c35d82 100644
 --- a/build/common.project.props
 +++ b/build/common.project.props
-@@ -23,7 +23,7 @@
-     <TargetFrameworksExeForSigning Condition=" '$(IsXPlat)' == 'true' ">$(NETCoreTargetFramework);netcoreapp5.0</TargetFrameworksExeForSigning>
-     <TargetFrameworksExeForSigning Condition="'$(DotNetBuildFromSource)' == 'true'">$(TargetFrameworksExe);net7.0</TargetFrameworksExeForSigning>
+@@ -26,7 +26,7 @@
```

Comment:
@crummel where is this coming from? If we are disabling TFM trimming, only the second set of changes is needed, i.e. a single `TargetFrameworksLibrary` change. My original change is: https://github.com/NuGet/NuGet.Client/commit/2da00ae1257a0d9262353230f7f6e34c57451f90

Reply:
OK - this looks good.

Reply:
There's no real change to your patch, just the context lines changed which caused it to fail.